### PR TITLE
Fix format issue in t/op/taint.t and add strict/warnings

### PR DESCRIPTION
**Pull request by atoomic**
Format was using '<'*5 instead of '<'x5 in multiple locations
also enforce strict & warnings to detect similar issues in the
future.

### CODE REVIEW

File: t/op/taint.t [hunk 1]
```diff
@@ -2300,17 +2305,17 @@ end
     is_tainted($^A, "tainted formline argument makes a tainted accumulator");
     $^A = "";
     isnt_tainted($^A, "accumulator can be explicitly untainted");
-    formline('@' .('<'*5) . ' | @*', 'hallo', 'welt');
+    formline('@' .('<'x5) . ' | @*', 'hallo', 'welt');
```
Comment (atoomic) [inline t/op/taint.t:2308]: notice the incorrect usage of `*5`

File: t/op/taint.t [hunk 2]
```diff
@@ -1655,7 +1657,7 @@ SKIP: {
 
     our $has_fcntl;
     BEGIN {
-	eval { require Fcntl; import Fcntl; };
+	eval { require Fcntl; Fcntl->import; };
```
Comment (atoomic) [inline t/op/taint.t:1660]: avoid indirect syntax

File: t/op/taint.t [hunk 3]
```diff
@@ -94,14 +95,14 @@ my $TAINT0;
 
 # This taints each argument passed. All must be lvalues.
 # Side effect: It also stringifies them. :-(
-sub taint_these (@) {
+sub taint_these :prototype(@) {
```
Comment (atoomic) [inline t/op/taint.t:98]: make the intent clear that this is a prototype

File: t/op/taint.t [hunk 4]
```diff
@@ -2167,7 +2171,7 @@ foreach my $ord (78, 163, 256) {
 
     is_tainted($string, "still tainted data");
 
-    my @got = split /[!,]/, $string;
+    @got = split /[!,]/, $string;
```
Comment (atoomic) [inline t/op/taint.t:2174]: multiple `my @got`... definitions in the same scope

File: t/op/taint.t [hunk 5]
```diff
@@ -2249,6 +2253,7 @@ foreach my $ord (78, 163, 256) {
 {
     for my $var1 ($TAINT, "123") {
 	for my $var2 ($TAINT0, "456") {
+        no warnings q(redundant);
```
Comment (atoomic) [inline t/op/taint.t:2256]: this test is using on purpose `sprint '%s', arg1, arg2` let's acknowledge it

File: t/op/taint.t [hunk 6]
```diff
@@ -2266,7 +2271,7 @@ foreach my $ord (78, 163, 256) {
 
 {
     use re 'taint';
-    "abc".$TAINT =~ /(.*)/; # make $1 tainted
+    my $x = "abc".$TAINT =~ /(.*)/; # make $1 tainted
```
Comment (atoomic) [inline t/op/taint.t:2274]: avoid void context

Comment (hvds) [reply t/op/taint.t:2274]: Given void context is perfectly sane here, it would be clearer to disable the warning than to bypass it.

**Pull request merged**